compiler/src/typed/oprint.re

```diff
@@ -549,7 +542,7 @@ and print_typargs = ppf =>
 and print_out_label = (ppf, (name, mut, arg)) =>
   fprintf(
     ppf,
-    "@[<v>@;<0 2>%s%s: %a,@]",
+    "@[<h>%s%s: %a@]",
```

Comment:
Was this one of those wierd box commas?

Reply:
Whoops, I think I removed this during one change and forgot to add it back. Good catch.